src/test/scala/tech/cryptonomic/nautilus/cloud/NautilusCloudStarterE2ETest.scala

```diff
@@ -240,18 +254,20 @@ class NautilusCloudStarterE2ETest
 
   "login endpoints" should {
 
-      "set a cookie after successful log-in" in {
+      "set an authorization header after successful log-in" in {
         // when
         val response = login("name@domain.com")
 
         // then
         response.body.right.value should matchJson(
           """{"header":{"payloadType":"REGISTERED"},"payload":{"userId":1,"userEmail":"name@domain.com","userRole":"user"}}"""
         )
-        response.cookies.headOption.value.name shouldBe "_sessiondata" // check if auth cookie named "_sessiondata" was set up
+        response.headers.toMap.get("Set-Authorization").value should endWith(
+          "xprovider~github&email~name%40domain.com&role~user&tosAccepted~true&id~1"
```

Comment:
I got a vague shiver down my spine... is this send over https like this...?

Reply:
yes, but also has the generated prefix

Reply:
Sure but looks like something that a man-in-the-middle might want to sniff out?

Did you consider jwt?

Reply:
makes no difference for man-in-the-middle if it's JWT or this - it will contain the same data, and always you'll be able to decode it and use it to perform actions if you get ahold of it. The prefix/signature is the thing that makes this/JWT safe as it will fail on decoding if those will be wrong.

Reply:
Let's continue the discussion offline because I think it's useful for me to understand, independently of the PR